### ISSUE TITLE
issue with harderore`s

### ISSUE THREAD

**Issue by hmbigburt** [Beyond-Reality/Beyond-Reality-Divergence#25]
for some reason when i craft the millstone the axle will not form neither will the wind veins fit on it properly, also cant get the sluice to work (will not process earth or sand into corresponding items). The problem exist on my ssp cant get them to work on creative either, don't think its me

**Comment by Dominance** [comment]
WindMill Multibock, Millstone and Pressure Packager work on my end how ever the sluice non-container does not seem to be working on my end ether. Have not tried the container version which is toggled on in the config. Maybe @Draco18s Can give this a look.

**Comment by Draco18s** [comment]
I will look at this in the morning. Are you on the most current version? I fixed a bug related to this not long ago.

**Comment by Draco18s** [comment]
Just looked at this. I am having no problems in dev with any of the referenced blocks.  Are you sure you're using _four_ axel pieces?
Edit: there is a problem with the (non-container) sluice.  Investigating.

**Comment by hmbigburt** [comment]
i have used 4 axel pieces and still didnt get the mill to form but since then have had probs with not being able to use the hardcore expansion book, it tells me i need to do /hqm quest to activate am treing a re-install at the minuite will let you know the results

**Comment by Draco18s** [comment]
Screenshot?

**Comment by Dominance** [comment]
Draco the multi blocks are working fine just the sluice is having issues. is giving me issues the later was user error. http://i.imgur.com/aPpc1UG.png

**Comment by Draco18s** [comment]
Sluice was fixed for 5.11, I'm still trying to figure out what's going wrong with hmbigburt's mills.

**Comment by Dominance** [comment]
Nothing is wrong with the mills. I have built about 100 since this issue came up. Could not find anything wrong with them myself.

**Comment by Draco18s** [comment]
Right, same here.  Which is why I want a screenshot from hmbigburt

**Comment by Dominance** [comment]
I did have another user suggest decreasing the milling time on ores. Other than that no other user had issues. I did have something else I recently fixed. Using Enviromine just for the hydration system. Everything is pretty much turned off. WIth linux it was auto creating a new default profile as a older version had the folder named Default instead of default. Which populated new default configs. It was turning on there physics system that I had disabled making it hard to build the windmill as the vanes would fall on there own. Again not your issue and was my own mistake :)
